onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -126,6 +126,40 @@ TEST_P(ModelTest, Run) {
+      {"softmax_cross_entropy_input_shape_is_NCd1_mean_weight_negative_ignore_index_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_mean_weight_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_ignore_index_3d", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_ignore_index_4d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_none_no_weight_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_ignore_index_4d", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3_sum_weight_high_ignore_index_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_3d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_3d", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_ignore_index_3d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_3d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none_weights_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_sum_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_3d", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3_sum_weight_high_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_sum", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none_weights", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_4d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1_mean_weight_negative_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_mean_weight", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_4d", "type error", {"onnx170"}},
 #endif
```

Comment:
Just curious: how about those `test_sce_*` test failures in onnx181/?

Reply:
I didn't see them failing.

Reply:
I changed them to use the pre-generated data in ONNX 1.8.1 release, it works well.